proto/api/v1/api.proto

```diff
@@ -543,7 +559,9 @@ service ConnectorService {
 
 message CreateConnectorRequest {
   Connector.Type type = 1;
+  // Plugin name
```

Comment:
We need more info here about builtin vs standalone plugins - this can be either an absolute path to a standalone plugin or the name of the builtin plugin.

Reply:
should talk about the difference between them? or just what the name would be in each case?